sui-execution/latest/sui-adapter/src/execution_engine.rs

```diff
@@ -281,6 +284,7 @@ mod checked {
     ) -> (
         GasCostSummary,
         Result<Mode::ExecutionResults, ExecutionError>,
```

Comment:
any plan to change this to `TimingResult` (or `ResultWithTiming`) eventually? or if not why not?

Reply:
They have to be split apart at some point, and this seemed like the most convenient point. I originally tried to keep them separate throughout the stack, but changing anything that returns `Result` to `(Result, T)` is very ugly - `?` stops working. Since this function already doesn't return a naked result it was easy to do the split here.